js/config-sample.json

```diff
@@ -20,9 +20,14 @@
             "y": 100
         },
         "UNITS_IN_MMOL": true,
-        "CALC_TREND": false
+        "CALC_TREND": false,
+        "ALWAYS_ON_TOP": true
+      },
+      "SHORTCUT": {
+        "ENABLED": true,
+        "COMBINATION": "Ctrl+Alt+G"
```

Comment:
The key combination should be the `key.code`d. String is not a best choice to set it.